### PR TITLE
Modulesync 0.12.2 & Release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
This fixes https://github.com/voxpupuli/puppet-graphite_powershell/issues/4 and makes https://github.com/voxpupuli/puppet-graphite_powershell/pull/5 obsolete.

**Comment by alexjfisher** [comment]
License in metadata.json doesn't match
https://github.com/voxpupuli/puppet-graphite_powershell/blob/master/manifests/init.pp#L3 or
https://github.com/voxpupuli/puppet-graphite_powershell/blob/master/LICENSE

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
-##2014-10-10 - Release 1.0.0
+## 2016-08-19 - Release 2.0.0
+
+* Drop of ruby1.8.7
+* Modulesync with latest Vox Pupuli changes
+* Fix: Hard coded path to config file in config class (issue #4)
```
Comment (igalic) [inline CHANGELOG.md]: alignment of licenses
we are an open source project, so that's noteworthy

Comment (bastelfreak) [inline CHANGELOG.md]: fixed

**Pull request merged**